src/AppBundle/Repository/PerformanceEventRepository.php

```diff
@@ -23,4 +23,13 @@ public function findByDateRangeAndSlug(\DateTime $fromDate, \DateTime $toDate, $
 
         return $query->execute();
     }
+
+    public function FindAllByDate(\DateTime $date)
+    {
+        $query = $this->createQueryBuilder('u')->where('u.dateTime LIKE :date')
```

Comment:
Yes 'like' )) because we have dateTime but we dont need to work with time, only with date))